plugins/dashboard/visualizer.go

```diff
@@ -113,8 +113,8 @@ func runVisualizer() {
 	})
 
 	if err := daemon.BackgroundWorker("Dashboard[Visualizer]", func(shutdownSignal <-chan struct{}) {
-		messagelayer.Tangle().Events.MessageAttached.Attach(notifyNewMsg)
-		defer messagelayer.Tangle().Events.MessageAttached.Detach(notifyNewMsg)
+		messagelayer.Tangle().MessageStore.Events.MessageStored.Attach(notifyNewMsg)
```

Comment:
I think we have to distinguish between a message being stored and attached. Or come up with a better name for it. 
This goes back to my earlier comment that we might need a flag to distinguish between a message being known to a node or actually being scheduled/booked.

Message stored: Could be any crap we get send from a neighbor. A node could exceed its rate (according to congestion control) and it will not be scheduled etc.

Message attached: Booking of message into local tangle. This happens **after** a message is scheduled, i.e., it has influence on consistency. (see #880)

Reply:
I prefer renaming `MessageAttached` to `MessageBooked`, align to `booked` in MessageMetadata.